cnames_active.js

```diff
@@ -110,6 +110,7 @@ var cnames_active = {
   "afilters": "dmytrohoi.github.io/afilters.js",
   "afmsavage": "afmsavage.github.io",
   "afterwind": "eedrxs.github.io/afterwind",
+  "agahi":"teneplaysofficial.github.io/agahi",
```

Comment:
```suggestion
  "agahi": "teneplaysofficial.github.io/agahi",
```
As the CI error notes, you are missing a space here.